moonstone/normalization/counts/random_selection.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+import numpy as np
+import pandas as pd
+
+from moonstone.filtering.basics_filtering import NamesFiltering
+from moonstone.normalization.base import BaseNormalization
+
+logger = logging.getLogger(__name__)
+
+
+class RandomSelection(BaseNormalization):
+    """
+    Randomly select a given number of counts (threshold) among all different items
```

Comment:
If I understand it right counts=reads? If so, I think it would maybe be more understandable using 'reads'

Reply:
Yes that's it. You get one count per one mapped reads (but this depends on the analysis this is why I am using counts which is more general).